cmake/kokkos_tribits.cmake

```diff
@@ -534,7 +534,7 @@ ELSE()
   CMAKE_PARSE_ARGUMENTS(PARSE
     ""
     ""
-    "HEADERS;SOURCES"
+    "HEADERS;SOURCES;NOINSTALLHEADERS"
```

Comment:
I don't think that change is needed.  Please undo.

Reply:
Are the parsed arguments used at all?

Reply:
It is, sadly required because I need to pass it in to tribits, but our function will throw a cmake error on an unrecognized argument in non-tribits mode :shrug:

The argument is used in non-tribits mode. This is yet another thing we will have to strike when we don't ahve to support tribits.

Reply:
Maybe not with multi value keyword args? I get:

```
CMake Error at cmake/kokkos_tribits.cmake:18 (if):
  if given arguments:

    "NOINSTALLHEADERS" <all the headers here>

  Unknown arguments specified
Call Stack (most recent call first):
  cmake/kokkos_tribits.cmake:127 (VERIFY_EMPTY)
  cmake/kokkos_tribits.cmake:542 (KOKKOS_INTERNAL_ADD_LIBRARY_INSTALL)
  algorithms/src/CMakeLists.txt:26 (KOKKOS_ADD_INTERFACE_LIBRARY)
```

Reply:
I suppose that is fine.  Thanks for getting to the bottom of it

Reply:
I still fail to see where the parsed arguments are used below. It seems that only `NAME` is relevant.

Reply:
Oh I see, yes I think we could actually remove the parse arguments. I'm not 100% sure why it is there.

Reply:
I'd prefer if we removed it

Reply:
I removed it entirely